src/containers/SearchPage/SearchPage.duck.js

```diff
@@ -131,12 +132,37 @@ export const searchListings = searchParams => (dispatch, getState, sdk) => {
       : {};
   };
 
-  const { perPage, price, dates, ...rest } = searchParams;
+  const availabilityParams = (datesParam, minDurationParam) => {
+    const dateValues = datesParam ? datesParam.split(',') : [];
+    const hasDateValues = datesParam && dateValues.length === 2;
+    const startDate = hasDateValues ? dateValues[0] : null;
+    const endDate = hasDateValues ? dateValues[1] : null;
+
+    const minDurationMaybe =
+      minDurationParam && Number.isInteger(minDurationParam) && hasDateValues
+        ? { minDuration: minDurationParam }
+        : {};
+
+    const tzId = config.custom.dateRangeLengthFilterConfig.searchTimeZoneId;
+
+    return hasDateValues
+      ? {
+          start: formatDateStringToTz(startDate, tzId),
+          end: getExclusiveEndDateWithTz(endDate, tzId),
+          availability: 'time-partial',
```

Comment:
Maybe these hard-coded values could use some comment text or link to api-reference.

We have added this to FTW-daily: https://github.com/sharetribe/ftw-daily/blob/master/src/containers/SearchPage/SearchPage.duck.js#L147